starter-code/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
+<link rel="stylesheet" type="text/css" href="styles.css">
+<script src="index.js"></script>
+  <title>Rivera Clone</title>
+</head>
+<body>
+  <div class="container-fluid"> 
+        <div class="row">
+            <div class="col-md-1"> </div>
+                <h1>RIVERA</h1>
+                <div class="col-md-5"> </div>
+                <nav class="list-inline">
+                        <a href="#"class="list-inline-item">Home</a>
+                        <a href="#"class="list-inline-item">About</a>
+                        <a href="#"class="list-inline-item">Sample Page</a>
+                        <a href="#"class="list-inline-item">Portfolio</a>
+                        <a href="#"class="list-inline-item">Blog</a>
+                        <a href="#"class="list-inline-item">Parent Page</a>
+                        <a href="#"class="list-inline-item">Contact</a>
+                      </nav>
```

Comment:
Repetition of the same class more than two times in a row is a signal that something needs to be changed. Check your CSS for a possible solution.